recipes/ophyd-async/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ophyd-async" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ophyd_async-{{ version }}.tar.gz
+  sha256: 57ea55a014c08fa463a5e514be6f4f93e33743e08811d69e73bfd06f1b1881e6
+
+build:
+  entry_points:
+    - ophyd-async = ophyd_async.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=64
+    - setuptools-scm >=6.2
+    - wheel
+    - pip
+  run:
+    - python >=3.10
+    - networkx >=2.0
+    - numpy <2.0.0
+    - packaging
+    - pint
+    - bluesky >=1.13.0a3
+    - event-model <1.21.0
+    # - p4p  # does not exist for py312
```

Comment:
Is it optional? How functional is the package without it?

Reply:
@ocepaf, thanks for looking into it and sorry for the delay with addressing your question. I've added a conditional inclusion of `p4p` and added a note with a relevant PR in the package's repo.